### PR TITLE
feat: [FTL-7647] User creation timestamp

### PR DESCRIPTION
**Pull request by linn-affinidi**
- [x] Add adminGetUserInfo() to common-libs/internal-api-clients
- [x] Expose user creation timestamp to client like `did`

### CODE REVIEW

File: sdk/core/src/CommonNetworkMember/NetworkMemberWithCognito.ts [hunk 1]
```diff
@@ -24,17 +24,20 @@ import { normalizeShortPassword } from '../shared/normalizeShortPassword'
 
 type UserDataWithCognito = ConstructorUserData & {
   cognitoUserTokens: CognitoUserTokens
+  createdDate: string
```
Comment (Volland) [inline sdk/core/src/CommonNetworkMember/NetworkMemberWithCognito.ts:27]: we dont need to add it as a field or keep this data . we could just use  a call to service .

Comment (Volland) [reply sdk/core/src/CommonNetworkMember/NetworkMemberWithCognito.ts:27]: Idea was to have it as a method on sdk

File: common-libs/internal-api-clients/src/services/KeyStorageApiService.ts [hunk 2]
```diff
@@ -82,6 +82,11 @@ export default class KeyStorageApiService {
     return this.client.AdminLogOutUser({ authorization: accessToken })
   }
 
+  async adminGetUserInfo({ accessToken }: { accessToken: string }) {
+    const result = await this.client.AdminGetUserInfo({ authorization: accessToken })
```
Comment (oleksiipiliugin) [inline common-libs/internal-api-clients/src/services/KeyStorageApiService.ts:86]: Just FYI
it is better to use auto-generated types here next time:
```
async adminGetUserInfo(params: GetParams<typeof clientMethods.AdminGetUserInfo>) {
    const result = await this.client.AdminGetUserInfo(params)
```

**Review by Volland — APPROVED** [review]

File: common-libs/internal-api-clients/src/services/KeyStorageApiService.ts (hunk 2 above)
Comment (linn-affinidi) [reply common-libs/internal-api-clients/src/services/KeyStorageApiService.ts:86]: sure, will change it next time.

**Pull request merged**